.github/workflows/e2e_spark.yaml

```diff
@@ -0,0 +1,68 @@
+name: E2E Spark Integration Test
+
+on:
+  # Post check in below branches
+  push:
+    branches:
+      - master
+  # PR triggered job
+  pull_request:
+jobs:
+  k8s-integration-tests:
+    name: "Run Spark (branch-3.3) / Volcano Integration test"
+    runs-on: ubuntu-20.04-spark
+    steps:
+
+    - name: Checkout current Volcano repository
+      if: github.event.inputs.volcano-branch==''
+      uses: actions/checkout@v2
+
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        repository: apache/spark
+        ref: branch-3.3
+        path: ${{ github.workspace }}/spark
+    - name: Install Java 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17.x
+    - name: start minikube
+      run: |
+        # Use pre-install minikube
+        minikube start
+    - name: Build lastest volcano images
+      run: |
+        eval $(minikube docker-env)
+        make TAG=latest update-development-yaml
+        make TAG=latest images
+        docker images | grep volcano
+        cat ./installer/volcano-development.yaml  | grep image:
+        kubectl apply -f ./installer/volcano-development.yaml
```

Comment:
Is thre any more dependency package needed to be installed such as kubectl?

Reply:
`kubectl`/`minikube` had been pre installed in self hosted runner, we might consider install specific version minikube/kubectl in future.

Reply:
To reduce deps on `kubectl`, we can use `minikube kubectl` instead.